src/ucp/wireup/wireup_cm.c

```diff
@@ -1176,47 +1193,20 @@ ucp_ep_server_init_priv_data(ucp_ep_h ep,  const char *dev_name,
                                  goto out;
                              });
 
-    tl_bitmap = ucp_ep_get_tl_bitmap(ep);
+    ucp_ep_get_tl_bitmap(ep, &tl_bitmap);
+    ucp_context_dev_tl_bitmap(worker->context, dev_name, &ctx_tl_bitmap);
 
     /* make sure that all lanes are created on correct device */
-    ucs_assert(UCS_BITMAP_IS_ZERO(
-            UCP_TL_BITMAP_AND_NOT(
-                    tl_bitmap, ucp_context_dev_tl_bitmap(worker->context,
-                                                         dev_name)),
-            UCP_MAX_RESOURCES));
+    ucs_assert(UCS_BITMAP_IS_ZERO(UCP_TL_BITMAP_AND_NOT(tl_bitmap,
+                                                        ctx_tl_bitmap),
+                                  UCP_MAX_RESOURCES));
```

Comment:
seems duplication of ucp_ep_create_to_worker_addr, can we push this to ucp_ep_get_tl_bitmap?